sdk/dataprotection/pom.xml

```diff
@@ -23,7 +23,7 @@
           <plugin>
             <groupId>org.jacoco</groupId>
             <artifactId>jacoco-maven-plugin</artifactId>
-            <version>0.8.5</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
+            <version>0.8.7</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
```

Comment:
Not sure how this has been passing build

Reply:
Maybe this pipeline (for dataprotection) has not been triggered ever since we upgraded jacoco version.